src/index.tsx

```diff
@@ -1,17 +1,34 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
```

Comment:
Does the import of the css index have to be removed? It seems like that might be an issue down the line.